lib/product-import.coffee

```diff
@@ -131,7 +131,7 @@ class ProductImport
         if @defaultAttributesService
           debug 'Ensuring default attributes'
           @_ensureDefaultAttributesInProducts(productsToProcess, queriedEntries)
-          .then () =>
+          .then ->
             Promise.resolve(queriedEntries)
         else
           Promise.resolve(queriedEntries)
```

Comment:
It is because of that async function which ensures default attributes in a product.